app/src/components/QuestPart.vue

```diff
@@ -9,22 +9,22 @@ api
   .get('/quests/helloworld')
   .then((res) => {
     console.log(res.data)
-    questData.value = JSON.stringify(res.data.content)
+    questData.value = JSON.stringify(res.data.instruction)
```

Comment:
I think we don't need v-html anymore since it's plain text.
```suggestion
    questData.value = res.data.instruction
```